experimental/emulated_vault/opt/emulated-vault/vault-server

```diff
@@ -0,0 +1,346 @@
+#! /usr/bin/python
+
+#
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+#
+#
+
+    
+import fnmatch
+import inspect
+import json
+import logging
+import os
+import re
+import ssl
+import sys
+
+import adapter.fs
+
+if sys.version_info >= (3, 0):
+    #python 3
+    from http.server import HTTPServer
+    from http.server import BaseHTTPRequestHandler
+    from configparser import ConfigParser
+    import urllib.parse as urlparse
+else:    
+    #python 2
+    from BaseHTTPServer import HTTPServer
+    from BaseHTTPServer import BaseHTTPRequestHandler
+    from ConfigParser import ConfigParser
+    import urlparse
```

Comment:
This all looks good, but if you wanted to use `print` instead of `sys.stderr.write` you could also add `from __future__ import print_function`. I think it should be fine either way, just wanted to be sure you were aware.

Reply:
Ack. Thanks